docs/tex/bib.bib

```diff
@@ -667,6 +667,14 @@ @article{marin2012approximate
 pages = {1167--1180}
 }
 
+@ARTICLE{Tipping99probabilisticprincipal,
```

Comment:
i recommend the citekey format `tipping1999probabilistic`, following google scholar.